src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3344,6 +3344,9 @@
   <data name="RFLCT_Targ_ITargMismatch" xml:space="preserve">
```

Comment:
Can this be removed?

Reply:
You actually suggested leaving the original string because it's used by runtime code :-)
https://github.com/dotnet/runtime/issues/97022#issuecomment-1896762339